src/clarity-angular/popover/signpost/_signposts.clarity.scss

```diff
@@ -24,8 +24,26 @@
                 height: $clr_baselineRem_1;
             }
 
-            &:hover, &.active {
+            &.active {
                 color: $clr-action-blue;
+                clr-icon {
+                    fill: $clr-action-blue;
+                }
+            }
+
+            &:hover {
```

Comment:
I don't understand why `&:hover, &.active` are separated into two different sections if they happen to have the same style.

Reply:
Right, I'll recombine them. I forget why I separated them.